docs/Installation-eks.md

```diff
@@ -0,0 +1,64 @@
+# Setup Kubernetes with Amazon EKS
+## Environments
+* EKS version: eks.3
+
+## Steps
+
+Basically, you just need to follow [the official document](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).  
+The diffrences are:
+
+- You do not need `Step 4: Launch a Guest Book Application`.
+- You need to use [our YAML file](https://github.com/drucker/drucker-parent/blob/master/config/amazon-eks-nodegroup.yaml) for CloudFormation to launch your worker nodes in `Step 3: Launch and Configure Amazon EKS Worker Nodes`.  
+  The details are in the section below.
+
+### Edit and Upload YAML file to launch your worker nodes.
+1. If you run `git clone` with SSH when you use drucker, replace `REPLACE_YOUR_OWN_SSH_KEY` with your own SSH private key in [YAML file](https://github.com/drucker/drucker-parent/blob/master/config/amazon-eks-nodegroup.yaml).  
+   You can skip this step if you use public repository and run `git clone` with HTTPS
+2. Follow the instructions instead of the official documemnt Step 3-5 and 3-6.
+    1. For **Choose a template**, select **Upload a template to Amazon S3.**
+    2. Select the YAML file you edited and choose **Next.**
+
+# Install Metrics Server
```

Comment:
It is a trivial comment.
"#" (h1 tag) must be used only once in the document. You must not use it in several times.